### PR TITLE
Add an assert to check that the pdf is uploaded as expected

### PR DESCRIPTION
**Pull request by willp-bl**

### CODE REVIEW

File: src/test/resources/small.pdf.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+%PDF-foo
```
Comment (idavidmcdonald) [inline src/test/resources/small.pdf.txt:1]: Do we want to name this `small.pdf` instead as that is maybe more realistic that a PDF file would just have the `pdf` extension (not that our API technically knows about that when it receives the data).

I guess I was just a bit surprised to see `.txt` on the end but it isn't a big deal

**Review by idavidmcdonald — APPROVED** [review]
Thanks for the particularly small PR, makes it quick to review.

Seems reasonable.

Maybe the filename for the file you added could be different, but I'll approve this because I'm fine if you feel strongly to keeping it as is. I don't feel strongly about my comment.

File: src/test/resources/small.pdf.txt (hunk 1 above)
Comment (willp-bl) [reply src/test/resources/small.pdf.txt:1]: I did originally use the `.pdf` extension but needed to add the `.txt` extension to be able to edit it more easily in my IDE.  Also the file likely won't open in a reader anyway as this is just designed to get past the code in the client that checks if a file is really a PDF.

Comment (idavidmcdonald) [reply src/test/resources/small.pdf.txt:1]: Happy with that. I'll merge this!

**Pull request merged**